nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/pom.xml

```diff
@@ -20,8 +20,8 @@ language governing permissions and limitations under the License. -->
     <packaging>jar</packaging>
 
     <properties>
-        <snmp4j.version>1.10.1</snmp4j.version>
-        <snmp4j-agent.version>1.3.1</snmp4j-agent.version>
+        <snmp4j.version>2.5.6</snmp4j.version>
+        <snmp4j-agent.version>2.5.3</snmp4j-agent.version>
```

Comment:
Released 22 June :) Good catch. I'll check and update. The agent is only used for the tests anyway so not too important. SNMP4J itself still seems to be at 2.5.6.